fivox/compartmentLoader.cpp

```diff
@@ -53,13 +53,7 @@ class CompartmentLoader::Impl
 
         helpers::addCompartmentEvents( morphologies, _report, output );
 
-        const float max = -60.f;
-        const float distance =
-                std::sqrt( std::abs( max ) / params.getMaxError( ));
-        LBINFO << "Computed cutoff distance: " << distance
-               << " with maximum event's value: " << max << std::endl;
-
-        output.setCutOffDistance( distance );
+        output.setCutOffDistance( params.getCutoffDistance( ));
```

Comment:
I don't think you need this here (or in any other loader) anymore, just set the cutoff distance here https://github.com/BlueBrain/Fivox/blob/master/fivox/eventSource.cpp#L57 and remove the setter if not used.